Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:bullseye-slim
```

Comment:
I wanted to use linuxserver/ffmpeg as a base image, but the current news process dependencies (`man` and `col` specifically) are a little difficult to get working in docker. If the news process changes, it might be worth changing the base image back there.

Reply:
This is currently the only way I could get anything reasonable to work but I will keep having a think about it

Reply:
Yeah I gave it a look, too and didn't see anything obvious.